cmd/feegrant.go

```diff
@@ -52,18 +55,27 @@ func feegrantConfigureBasicCmd(a *appState) *cobra.Command {
 			if len(args) > 1 {
 				granterKeyOrAddr = args[1]
 			} else if prov.PCfg.FeeGrants != nil {
-				granterKeyOrAddr = prov.PCfg.FeeGrants.GranterKey
+				granterKeyOrAddr = prov.PCfg.FeeGrants.GranterKeyOrAddr
 			} else {
 				granterKeyOrAddr = prov.PCfg.Key
 			}
 
+			externalGranter := false
+
 			granterKey, err := prov.KeyFromKeyOrAddress(granterKeyOrAddr)
 			if err != nil {
-				return fmt.Errorf("could not get granter key from '%s'", granterKeyOrAddr)
+				externalGranter = true
+			}
+
+			if externalGranter {
+				_, err := prov.DecodeBech32AccAddr(granterKeyOrAddr)
+				if err != nil {
+					return fmt.Errorf("an unknown granter was specified: '%s' is not a valid bech32 address", granterKeyOrAddr)
+				}
 			}
 
 			if delete {
-				fmt.Printf("Deleting %s feegrant configuration\n", chain)
+				a.log.Info("deleting feegrant configuration", zap.String("chain", chain))
```

Comment:
nit: logs should begin with an uppercase letter https://github.com/cosmos/relayer/blob/main/docs/logging.md#usage